test/unit/mixins/child-apps.spec.js

```diff
@@ -389,4 +389,44 @@ describe('ChildAppMixin', function() {
       expect(spy.returned(this.myChildApp)).to.be.true;
     });
   });
+
+  describe('restartWithParent', function() {
```

Comment:
I believe these tests likely belong in this file: https://github.com/RoundingWellOS/marionette.toolkit/blob/master/test/unit/app-lifecycle-options.spec.js